entrypoint.sh

```diff
@@ -17,14 +30,19 @@ else
   VERSION='patch';
 fi
 
+# What is this? 41898282+github-actions[bot]@users.noreply.github.com
+# Configuring ... wat?
+# Bumping and pushing
+# Why pulling?
 git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
 git config --global user.name "Github CI"
 bump2version --config-file .bumpversion.cfg "${VERSION}"
 git push https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
 git push https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git --tags
 git pull
 
-NEW_TAG="$(git describe)"
+# Fetching new tag ?? and releasing it using the PAT of whom? 
```

Comment:
this is using PAT in the workflow which you install inside repository you install this on (in settings/secrets). I put my own PAT in the organisation globally (so all releases will be authored by me, until i get hold of the clingen-sthlm)